### DISCUSSION TITLE
a Social media app for donation

### DISCUSSION THREAD

**Discussion by ehsaaanes** [WeAreMahsaAmini/Core#163] [Introductions]
سلام به همگى، مرسى سياوش جان كه اين محيطو فراهم كردين.

**Comment by AnonymousMaximus2** [comment]
من یک پیشنهادی برای بهتر شدن این پروژه شما در لینک زیر داده ام تا از نظر مالی و حمایت بین المللی بشه بهتر رشدش داد:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by ghost** [comment]
I would think about the distribution channel for this app, it is very difficult to do with all the app stores blocked. Also, Twitter and Instagram already cover these functionalities, and most people already have them, why create a new social media app?

**Reply by ehsaaanes** [reply]
Thanks for your reply, The mission of this social media is different from Twitter or Instagram . Here is to find people in need, see their problems, follow their art, listen to their music and podcast and so on. It surely still needs changes to achieve its goal but mission and vision is more like gofundme in the shape of social media. What did you mean by app stores blocked?

**Reply by ghost** [reply]
I mean Google Play is blocked/filtered/inaccessible, and you cannot put this app on Cafebazaar or Myket or other Iranian app stores. So how are you going to get people to download and install this app?

**Reply by ehsaaanes** [reply]
I believe the way people download other apps, they would download it too. As I know they use VPN to directly download feom app store. 
To me, the most important thing is that the idea would work to spend my time and money on it. Would you use this app for yourself?

**Comment by AnonymousMaximus2** [comment]
Who will be responsible for Anti Money laundering and legal aspects of this idea ?

You never can not do successful job until you take serious my idea below link to establish your not for profit organization:


you need at least 1 person can tell you about difference between legal acts and illegal acts, copyrights, Anti money laundering , tax, having separate bank account, ..... responsible persons for collecting money from people and spending money for elligible people.

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by samgholami** [comment]
Hello,
Although I like the idea, I do not believe it is doable in the current Iran situation. People need to trust this app and building this needs time. On the other hand, you need a big group of volunteers inside Iran to at least check the accuracy of the news. provide security, legal issues...( as AnonymousMaximus2 mentioned as well).
In my opinion, you can focus on gathering music, art, podcast, events, and... that can bring hope to people.  I also can suggest a section that introduces basic human rights and women's rights, even after victory people need to know their rights and need to protect that.
Let me know your thought and I would be glad to help in the design aspect and UX research.

**Reply by ehsaaanes** [reply]
Hi, thanks for your reply, great idea about human rights. I will add it as an API in feeds screen. I need help for designing it Figma or Adobex. Can you do it? if yes, where can I send the screens to you?

**Comment by hiddify-com** [comment]
یکی از فیچرهای لازم اینه که آدمها را شناسایی کنه و چهرشون رو محو کنه قبل از ارسال ویدیو یا عکس
ولی مشکل اساسی تر اینه که از فیلترینگ باید عبور کنیم و این دردسر داره

**Reply by ehsaaanes** [reply]
مرسى از پيامتون، تقريباً همه شبكات اجتماعى فيلترن ولى مردم استفاده ميكنن. هدف اپ رسوندن پول به آدماست، همه فيچرا درخدمت ارائه شدن آدماست واسه گرفتن كمك مالى. 

تو ساخت ميتونين كمكى كنين؟ اگه آره لطفاً بگين چه كمكى؟

**Reply by hiddify-com** [reply]
چه جوری میتونی وریفای کنی که آدم صحیحی را دارید کمک میکنید و سایبری نیست؟

**Reply by ehsaaanes** [reply]
قرار نيست اپ اينكارو كنه، تشخيصش با كاربريه كه ميخواد كمك كنه

**Comment by ghost** [comment]
Hi Ehsan. The idea is interesting but I, like others, have concerns about legal issues and also users' trust in the product.

**Reply by ehsaaanes** [reply]
Hi, thanks for your reply, like youtube, users of this app inside Iran need to find someone abroad to receive their donations. Also, this social media is not just for Iranians and could be usful in all countries.
Btw, If you'd like to collaborate on building it please let me know how you can help. Regards

**Comment by bardiakeyvani** [comment]
I think this is a great idea and like to help out. Have you thought about allowing the app to run on Tor network directly to prevent any cencership?

**Reply by ehsaaanes** [reply]
Hi, Thanks for your reply, I believe apps are more likely to be viral and easier for daily basis usage, but i'm open for any new ideas.  Please lemme know how you can help?

**Comment by ghost** [comment]
Hi Ehsan, I'd like to help with UX/UI. Let me know how to collaborate.

**Reply by ehsaaanes** [reply]
Hi, thanks for your reply, please send me a message on telegram: @ehsaaanes

**Reply by samgholami** [reply]
I would be happy to work with you and Masou on this.

**Reply by ehsaaanes** [reply]
Thank you for your interest, please text me on telegram: @ehsaaanes

**Comment by ghost** [comment]
Can I have your details to be able to send a message on Telegram please?

Thanks!

Masou

**Reply by ehsaaanes** [reply]
yes, please check my Facebook profile, thanks

https://www.facebook.com/ehsan.esmal

**Comment by ehsaaanes** [comment]


**Reply by ghost** [reply]
Shoma kheili asabani hasti. Shahsidam be harf zadanet, mage kuasi beset bedehi dare?
